src/hypertable.h

```diff
@@ -37,7 +37,7 @@ enum
 	HypertableInternalCompressionTable = 2,
 };
 
-#define TS_HYPERTABLE_HAS_COMPRESSION(ht) ts_hypertable_has_compression(ht)
+#define TS_HYPERTABLE_HAS_COMPRESSION_TABLE(ht) ts_hypertable_has_compression_table(ht)
```

Comment:
why not to use function in such case without macro?

Reply:
Using a macro keeps it consistent with the other compression related macros like TS_HYPERTABLE_HAS_COMPRESSION_ENABLED